changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## NEXT
+
+- Fixed a bug that would allow the addition of b--a and b->a edges to a graph with an existing undirected a--b edge in
```

Comment:
Actually, your fix is more generally than this, right?

Might be better to rephrase it so that reverse edges (for non-directed edge types) can simply no longer be specified

Reply:
I would maybe recommend something like this:

- Fixed a bug in the `cai_causal_graph.causal_graph.CausalGraph.add_edge` method that would allow the addition of an edge between `b` and `a` even when the reverse edge, i.e. between `a` and `b` was already specified.